### PR TITLE
Logging changed

### PR DESCRIPTION
**Pull request by bulicamihai**

### CODE REVIEW

**Comment by bulicamihai** [comment]
[New Content Monitoring: output log lines for each event/app](https://trello.com/c/Bvwc5JRH/197-new-content-monitoring-output-log-lines-for-each-event-app)

File: resources/delete.go [hunk 1]
```diff
@@ -24,14 +24,15 @@ func DeleteContent(mongo db.DB) func(writer http.ResponseWriter, req *http.Reque
 		collectionID := mux.Vars(r)["collection"]
 		resourceID := mux.Vars(r)["resource"]
 		tid := obtainTxID(r)
+		contentTypeHeader := extractContentTypeHeader(r, tid, resourceID)
 
 		if err := connection.Delete(collectionID, resourceID); err != nil {
 			msg := "Deleting from mongoDB failed"
-			logger.WithTransactionID(tid).WithUUID(resourceID).WithError(err).Error(msg)
+			logger.WithMonitoringEvent("SaveToNative", tid, contentTypeHeader).WithUUID(resourceID).WithError(err).Error(msg)
 			http.Error(w, fmt.Sprintf(msg+"\n%v\n", err), http.StatusInternalServerError)
```
Comment (MihaiSorinMoisa) [inline resources/delete.go]: Change `fmt.Sprintf(msg+"\n%v\n", err)` to `fmt.Sprintf("%s\n%v\n", msg, err)`.

File: resources/write.go [hunk 2]
```diff
@@ -29,15 +29,15 @@ func WriteContent(mongo db.DB) func(w http.ResponseWriter, r *http.Request) {
 		inMapper, err := mapper.InMapperForContentType(contentTypeHeader)
 		if err != nil {
 			msg := "Unsupported content-type"
-			logger.WithMonitoringEvent("NativeSave", tid, contentTypeHeader).WithUUID(resourceID).WithError(err).Error(msg)
+			logger.WithMonitoringEvent("SaveToNative", tid, contentTypeHeader).WithUUID(resourceID).WithError(err).Error(msg)
 			http.Error(w, fmt.Sprintf(msg+":\n%v\n", err), http.StatusBadRequest)
```
Comment (MihaiSorinMoisa) [inline resources/write.go]: Change `fmt.Sprintf(msg+"\n%v\n", err)` to `fmt.Sprintf("%s\n%v\n", msg, err)`.

File: resources/write.go [hunk 3]
```diff
@@ -29,15 +29,15 @@ func WriteContent(mongo db.DB) func(w http.ResponseWriter, r *http.Request) {
...
 			return
 		}
 
 		content, err := inMapper(r.Body)
 		if err != nil {
 			msg := "Extracting content from HTTP body failed"
-			logger.WithMonitoringEvent("NativeSave", tid, "").WithUUID(resourceID).WithError(err).Error(msg)
+			logger.WithMonitoringEvent("SaveToNative", tid, "").WithUUID(resourceID).WithError(err).Error(msg)
 			http.Error(w, fmt.Sprintf(msg+":\n%v\n", err), http.StatusBadRequest)
```
Comment (MihaiSorinMoisa) [inline resources/write.go]: Change `fmt.Sprintf(msg+"\n%v\n", err)` to `fmt.Sprintf("%s\n%v\n", msg, err)`.

File: resources/write.go [hunk 4]
```diff
@@ -46,11 +46,11 @@ func WriteContent(mongo db.DB) func(w http.ResponseWriter, r *http.Request) {
 
 		if err := connection.Write(collectionID, wrappedContent); err != nil {
 			msg := "Writing to mongoDB failed"
-			logger.WithMonitoringEvent("NativeSave", tid, "").WithUUID(resourceID).WithError(err).Error(msg)
+			logger.WithMonitoringEvent("SaveToNative", tid, "").WithUUID(resourceID).WithError(err).Error(msg)
 			http.Error(w, fmt.Sprintf(msg+":\n%v\n", err), http.StatusInternalServerError)
```
Comment (MihaiSorinMoisa) [inline resources/write.go]: Change `fmt.Sprintf(msg+"\n%v\n", err)` to `fmt.Sprintf("%s\n%v\n", msg, err)`.

**Pull request merged**